src/metabase/models/session.clj

```diff
@@ -104,3 +107,12 @@
   (when-not (public-settings/enable-password-login)
     (throw (ex-info (str (tru "Password login is disabled for this instance.")) {:status-code 400})))
   ((get-method create-session! :sso) session-type user device-info))
+
+(defn cleanup-sessions
```

Comment:
```suggestion
(defn cleanup-sessions!
```

Reply:
Fixed in 53481 since this was already merged